app/models/agents/rss_agent.rb

```diff
@@ -105,6 +106,10 @@ def validate_options
         errors.add(:base, "Please provide 'expected_update_period_in_days' to indicate how many days can pass without an update before this Agent is considered to not be working")
       end
 
+      if options['max_ids'].present? && options['max_ids'].to_i < 1
```

Comment:
Since you are using the interpolated value of `max_ids` below, we should also use `interpolated` in the validation.